config/config.exs

```diff
@@ -0,0 +1,3 @@
+use Mix.Config
+
+config :elixir, :time_zone_database, Tzdata.TimeZoneDatabase
```

Comment:
Should we add a note on readme to add this setup when using the library?

Reply:
não precisei adicionar isso no hermes, já usou o que está configurado na própria lib